src/nexusrpc/handler/_cancellation.py

```diff
@@ -0,0 +1,52 @@
+from abc import ABC, abstractmethod
+from typing import Optional
+
+
+class OperationTaskCancellation(ABC):
```

Comment:
This should be exposed out of `__init__.py` (or just put in another file and have it exposed in init out of that file since single-class files are not that common)